modules/op-adding-triggers.adoc

```diff
@@ -81,18 +81,46 @@ spec:
           claimName: source-pvc
 ----
 
-. Create the `TriggerTemplate`:
+. Create the `TriggerTemplate` resource:
 +
 ----
 $ oc create -f <triggertemplate-yaml-file-name.yaml>
 ----
 +
-Alternatively, you can create the `TriggerTemplate` directly from the `pipelines-tutorial` Git repository:
+Alternatively, you can create the `TriggerTemplate` resource directly from the `pipelines-tutorial` Git repository:
 +
 ----
 $ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/release-tech-preview-2/03_triggers/02_template.yaml
 ----
 
+. Copy the contents of the following sample `Trigger` YAML file and save it:
++
+[source,yaml]
+----
+apiVersion: triggers.tekton.dev/v1alpha1 <1>
+kind: Trigger <2>
+metadata:
+  name: vote-trigger <3>
+spec:
+  serviceAccountName: pipeline <4>
+  bindings:
+    - ref: vote-app <5>
+  template:
+     name: vote-app
+----
+
+. Create the `Trigger` resource:
++
+----
+$ oc create -f <trigger-yaml-file-name.yaml>
+----
++
+Alternatively, you can create the `Trigger` resource directly from the `pipelines-tutorial` Git repository:
++
+----
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/03_trigger.yaml
```

Comment:
We cannot reference master branch... https://github.com/openshift/pipelines-tutorial/blob/release-tech-preview-3/03_triggers/03_trigger.yaml

BTW, we still reference branch `release-tech-preview-2` instead of `release-tech-preview-3` in other places. Would it be possible to introduce a variable that we would simply update in every release?

Reply:
The variable already exists in the generic commands. This is a direct command specifically for the getting started tutorial example to work, so I am not sure a variable would help. If we add it, we will force users to go to the GH repo to search for the appropriate branch. Which is why I have a created a recurring task in Jira for each release to update these.

Reply:
I don't follow your comment. My suggestion is to introduce a variable that would contain branch name and then in every release you would just update this one value. All links would be rendered in such a way that they are clickable like they are today and for end users it would be transparent. User would not be forced to go to the GH repo and search appropriate branch.

We already use variable `{product-title}` that renders as `OpenShift Container Platform` therefore I assume that it's possible to introduce new variables.

Reply:
Ah, alright, I mistook you to mean end user replaceable variable. This is a good solution. I will do that.

Reply:
@ppitonak I added the attribute, but adding attributes in commands and YAML snippets does not render.  Please see screenshot and the preview.
![Screenshot from 2020-11-24 23-37-30](https://user-images.githubusercontent.com/16647625/100134591-72ca9b80-2eae-11eb-9d3a-e3fec2172def.png)
I will check with folks in the OCP docs team too just to confirm.